src/main/scala/sparkz/core/network/SendingStrategy.scala

```diff
@@ -33,17 +31,6 @@ case object BroadcastTransaction extends SendingStrategy {
   }
 }
 
-case class BroadcastBlock(settings: NetworkSettings) extends SendingStrategy {
-  val maxBlockBroadcastPeers: Int = settings.maxForgerConnections +
```

Comment:
sorry maybe my comment was not clear: now you removed too much code!
in order to broadcast the block to every peer but with priority to forgers we need to return a list of all connections but with all the forgerPeers at the beginning of the list

Reply:
Fixed, my bad for not understanding the idea